kolibri/plugins/setup_wizard/api.py

```diff
@@ -115,18 +127,49 @@ def provisiondevice(self, request):
 
         return Response({})
 
+    @decorators.action(methods=["post"], detail=False)
+    def listfacilitylearners(self, request):
+        """
+        If the request is done by an admin user  it will return a list of the users of the
+        facility
+
+        :param baseurl: First part of the url of the server that's going to be requested
+        :param facility_id: Id of the facility to authenticate and get the list of users
+        :param username: Username of the user that's going to authenticate
+        :param password: Password of the user that's going to authenticate
+        :return: List of the learners of the facility.
+        """
+        facility_info = getusersinfo(request)
+        user_info = facility_info["user"]
+        roles = user_info["roles"]
+        admin_roles = (user_kinds.ADMIN, user_kinds.SUPERUSER)
+        if not any([role in roles for role in admin_roles]):
+            raise PermissionDenied()
+        students = [u for u in facility_info["users"] if not u["roles"]]
+        return Response({"students": students, "admin": facility_info["user"]})
+
 
 class SetupWizardFacilityImportTaskView(FacilityTasksViewSet):
     """
     An open version of FacilityTasksViewSet for the purposes of managing the
     import-facility task during setup
     """
 
-    permission_classes = (HasPermissionDuringSetup,)
+    permission_classes = [HasPermissionDuringSetup | HasPermissionDuringLODSetup]
```

Comment:
We are inheriting `FacilityTasksViewSet` so that we can send request to `create` API during setup process, right? Did I get it correct? 

If yes then we will not need to inherit `FacilityTasksViewSet` after merging #8303.

Reply:
Yep, thanks, @vkWeb,   I bear it in mind. As soon as #8303 is merged I will be really happy to remove this subclassing but in the meantime we need this to be working to be able to test the sync processs. 
I'll try to prepare a PR this week with the changes needed so you can check it and we can merge it right after #8303 is merged too.

Reply:
Sir, that sounds like a good plan!